examples/basic_usage/variables.tf

```diff
@@ -0,0 +1,33 @@
+# ------------------------------------------------------------------------------
+# Required parameters
+#
+# You must provide a value for each of these parameters.
+# ------------------------------------------------------------------------------
+
+# ------------------------------------------------------------------------------
+# Optional parameters
+#
+# These parameters have reasonable defaults.
+# ------------------------------------------------------------------------------
+
+variable "ami_owner_account_id" {
+  description = "The ID of the AWS account that owns the AMI, or \"self\" if the AMI is owned by the same account as the provisioner."
```

Comment:
You need a type here.

Reply:
Good eye!  Removed in https://github.com/cisagov/skeleton-tf-module/pull/39/commits/0dd4c0ce78af5acc7eff7ddfdde85209f8ff94ca.

Reply:
Thanks- fixed in https://github.com/cisagov/skeleton-tf-module/pull/39/commits/1b84540513882933410a958e9205fb11cf8094dc.